frontend/src/metabase/admin/permissions/selectors/data-permissions/groups.ts

```diff
@@ -1,25 +1,17 @@
 import { createSelector } from "@reduxjs/toolkit";
 import _ from "underscore";
-import { assoc } from "icepick";
 
 import Groups from "metabase/entities/groups";
 import { Group } from "metabase-types/api";
-import {
-  getGroupNameLocalized,
-  isAdminGroup,
-  isDefaultGroup,
-} from "metabase/lib/groups";
+import { isAdminGroup, isDefaultGroup } from "metabase/lib/groups";
```

Comment:
Revert the code to where it is before https://github.com/metabase/metabase/pull/33035. The reason is explained in the comment in `frontend/src/metabase/admin/permissions/selectors/data-permissions/group-sidebar.ts`